src/hotspot/cpu/riscv/c1_LIRAssembler_riscv.cpp

```diff
@@ -1372,7 +1372,7 @@ void LIR_Assembler::ic_call(LIR_OpJavaCall* op) {
 
 void LIR_Assembler::emit_static_call_stub() {
   address call_pc = __ pc();
-  assert((__ offset() % 4) == 0, "bad alignment");
+  __ assert_alignment(call_pc);
```

Comment:
As `assert_alignment` is defined as a  static method, there is no need for `__`. `MacroAssembler::assert_alignment(call_pc)` would be better.

Reply:
Reasonable; all done.